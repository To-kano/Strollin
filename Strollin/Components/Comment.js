import {
    Button, Image, View, StyleSheet, Text, ScrollView, FlatList
  } from 'react-native';
  import React from "react";
  
  function Comment(props)  {
  
    console.log("props = ", props);
    return (
      <View style={{
        justifyContent: 'space-around', flex: 1, marginTop: 20, marginHorizontal: '10%', backgroundColor: 'rgba(255,255,255, 0.9)', borderRadius: 20, width: '80%'
      }}
      >
        <Text style={[{ fontSize: 35 }]}>{props.pseudo}</Text>
        <Text></Text>
        <Text style={[{ fontSize: 25 }]}>{props.comment}</Text>
        <Text style={[{ fontSize: 25 }]}>Note : {props.note}</Text>
      </View>
    );
  }
  
  export default Comment;
  
  const styles = StyleSheet.create({
    cont: {
      marginTop: '5%',
      flexDirection: 'column',
      justifyContent: 'flex-start',
      alignItems: 'center',
      flex: 0.1,
      backgroundColor: '#FFC300'
    },
    img: {
      flexDirection: 'column',
      justifyContent: 'flex-start',
      alignItems: 'center',
      flex: 0.1,
      backgroundColor: 'red',
      width: '80%'
    },
    whiteBox: {
      justifyContent: 'space-between',
      flex: 1,
      marginTop: 20,
      marginHorizontal: '10%',
      backgroundColor: 'rgba(255,255,255, 0.9)',
      borderRadius: 20,
      textAlign: "left",
      width: '80%'
    }
  
  });
  