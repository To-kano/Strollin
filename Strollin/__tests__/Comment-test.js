import React from 'react';
import {shallow} from 'enzyme';
import Comment from '../Components/Comment';

const dataTest = {
    name: 'test',
    budget: '10',
    period: '10 am to 10 pm',
    destinations: [
        '1',
        '2',
        '3',
    ]
}

describe('Comment', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<Comment data={dataTest}/>)
            expect(component).toMatchSnapshot()
        });
    });
});