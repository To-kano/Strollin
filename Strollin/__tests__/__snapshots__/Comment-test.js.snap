// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
